pkg/util/resource-recommend/types/conditions/conditions.go

```diff
@@ -23,6 +23,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/kubewharf/katalyst-api/pkg/apis/recommendation/v1alpha1"
+
```

Comment:
remove the blank line to obey the import rule of three segements: the native imports, the other repos' imports and own imports